### PR TITLE
feat: migrate and apply crossRegionPresignedUrlMiddleware

### PR DESCRIPTION
**Pull request by srchase**
Migrates and applies crossRegionPresignedUrlMiddleware for RDS clients

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/middleware-sdk-rds/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (AllanFly120) [inline packages/middleware-sdk-rds/LICENSE]:
```suggestion
   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

**Comment by AllanFly120** [comment]
Merge it when query protocol is ready and have tested this agains RDS client.

File: packages/middleware-sdk-rds/LICENSE (hunk 1 above)
Comment (srchase) [reply packages/middleware-sdk-rds/LICENSE]: updated.

**Comment by trivikr** [comment]
The code was reverted using force-push, and re-added as we were testing what happens when remote gets updated during release.
* re-added commit while release was in progress https://github.com/aws/aws-sdk-js-v3/commit/3e24d71dcd129322394879a6de9e9dad83d276b4
* merge commit https://github.com/aws/aws-sdk-js-v3/commit/85fb6b620d59a62a9ea9f2770803ffcacceb8bff

Release https://github.com/aws/aws-sdk-js-v3/releases/tag/v1.0.0-alpha.6

**Pull request merged**